libs/blocks/quiz/quiz.css

```diff
@@ -873,8 +610,366 @@ TODO: Need to check if we can skip this somehow.
+    text-align: right;
+  }
+
+  .quiz-container.split .text .icon-area img {
+    max-height: 56px;
+    max-width: 212px;
+  }
+
+  .quiz-container.split.large .text .icon-area img {
+    max-height: 80px;
+  }
+
+  .quiz-container.split.row-reversed .media.bleed {
+    right: auto;
+    left: 0;
+  }
+
+  .quiz-container.split.row-reversed .foreground.container {
+    justify-content: flex-end;
+  }
+
+  .quiz-container.split .media.bleed picture,
+  .quiz-container.split .media.bleed video {
+    height: 100%;
+    object-fit: fill;
+  }
+
+  .quiz-container.split.large .text {
+    margin: 0;
+  }
+
+  .quiz-container.split.small .media img,
+  .quiz-container.split.large .media img,
+  .quiz-container.split .media img,
+  .quiz-container.split.small .media video,
+  .quiz-container.split.large .media video,
+  .quiz-container.split .media video {
+    max-height: initial;
+  }
+
+  .quiz-container.split .button-justified-mobile {
+    width: auto;
+  }
+}
+
+@media screen and (min-width: 899px) and (max-width: 1221px) {
+  .quiz-page .gnav {
+    padding: 0 20px;
+  }
+}
+
+@media screen and (max-width: 899px) {
+  .quiz-container .consonant-OneHalfCard-text {
+    margin: 0;
+  }
+}
+
+@media screen and (min-width: 900px) {   
+  .quiz-container .card .consonant-OneHalfCard-title {
+    display: block;
+  }
+
+  .quiz-container .consonant-OneHalfCard-text,
+  .quiz-container .consonant-OneHalfCard-title {
+    text-align: center;
+    font-size: 14px;
+    line-height: 1.375rem;
+    display: block;
+    visibility: visible;    
+  }  
+  .quiz-container .card.selected .consonant-OneHalfCard-text {
+    font-weight: normal;
+  }
+
+  .card .consonant-OneHalfCard-title {
+    display: block;
+  }
+
+  .quiz-container.split.one-third {
+    display: flex;
+    justify-content: center;
+  }
+
+  .quiz-container.split.one-third.row-reversed .foreground.container {
+    justify-content: flex-end;
+  }
+
+  .quiz-container.split.one-third .media.bleed {
+    position: absolute;
+    width: calc(var(--grid-container-width) * 0.6667 + (100vw - var(--grid-container-width)) / 2);
+    right: 0;
+    height: 100%;
+  }
+
+  .quiz-container.split.one-third .foreground.container .text,
+  .quiz-container.split.one-third.large .foreground.container .text {
+    max-width: calc(33.3334% - var(--grid-column-width));
+  }
+
+  .quiz-container.split.one-third .text .icon-area img {
+    max-height: 56px;
+    max-width: 159px;
+  }
+
+  .quiz-container.split.one-third.row-reversed .media-credit {
+    text-align: left;
+  }
+
+  .quiz-container.split.one-third .media-credit {
+    text-align: right;
+  }
+
+  .quiz-container .consonant-OneHalfCard {
+    padding-bottom: 0;
+  }
+  .quiz-container .quiz-options-container {
+    display: grid;
+  }
+
+  .milo-card-wrapper {
+    padding: 0;
+  }
+
+  .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option {
+    background-color: fafafacc;
+    min-width: 9.625rem;
```

Comment:
This is the same value as `width`, it shouldn't be necessary

Reply:
As implemented it is needed, but I will look at refactoring this in the next related ticket. Thanks for all these callouts, I appreciate it.